blueprints/acceptance-test/qunit-rfc-232-files/tests/acceptance/__name__-test.js

```diff
@@ -1,6 +1,6 @@
 import { module, test } from 'qunit';
 import { visit, currentURL } from '@ember/test-helpers';
-import { setupApplicationTest } from 'ember-qunit';
+import setupApplicationTest from '<%= dasherizedPackageName %>/tests/helpers/setup-application-test';
```

Comment:
Should use `<%= testFolderRoot %>` here, same as [here](https://github.com/emberjs/ember.js/blob/master/blueprints/acceptance-test/qunit-files/tests/acceptance/__name__-test.js#L2), no?

Reply:
I'm wondering if we would need to detect if the file actually exists and if not fall back to the old behavior? 🤔

Reply:
Oof, that would be a pain...I think that would involve `import require from 'require'` and using `require.has()` to import rather than ES6 imports, right?

Reply:
Oh, or you mean detecting at blueprint execution time (rather than runtime)?

Reply:
yes, that is what I meant. I roughly remember that we already did something like that somewhere for the `destroy-app.js` file 🤔